lib/modifiers/flatten.js

```diff
@@ -0,0 +1,60 @@
+var _ = require('lodash');
+var through = require('through2');
+
+/**
+ * Returns a new stream that flattens all arrays passing through by one level. All non-array
+ * items will be passed through as-is.
+ *
+ * @method
+ * @static
+ * @since    1.5.0
+ * @category Util
```

Comment:
The category should be "Modifiers". I just realized `batch` is also wrong and am committing a fix for it. "Util" is used for the imported stuff like `split`, `pipeline` and `through`.

At some point I want to figure out a better system since "Modifiers" is massive and could conceivably be much more massive than it is now.